Models/Employee/Permission.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace PositronAPI.Models.Employee
+{
+    public enum Permission
+    {
+        None,
+        View,
+        Edit
+    }
+}
```

Comment:
Permission seems a weird thing to bring over form the spec sheet, since we already have Role as an enum maybe just do that if needing to handle Write, Read, View permissions? I'd suggest removing it all together and just keeping the Role enum